coffee_github/src/repository.rs

```diff
@@ -97,14 +97,20 @@ impl Github {
                             debug!("found plugin configuration: {}", conf_str);
 
                             let conf_file = serde_yaml::from_str::<Conf>(&conf_str).unwrap();
+                            let conf_lang = (&conf_file.plugin.lang).to_owned();
+                            plugin_lang = match conf_lang.as_str() {
+                                "py" => PluginLang::Python,
+                                "go" => PluginLang::Go,
+                                "rs" => PluginLang::Rust,
+                                "dart" => PluginLang::Dart,
+                                "js" => PluginLang::JavaScript,
+                                "ts" => PluginLang::TypeScript,
+                                _ => continue,
```

Comment:
```suggestion
                                _ => CoffeError::new(1, format!("language {conf_lang} not supported"),
```

This is not a continue, but this is an error because this mean that we do not know the language

Reply:
I refactored the match expression to be able to return a `CoffeeError`.